public/js/dataset-summary.js

```diff
@@ -48,7 +48,7 @@ $(document).ready(function() {
     deleteDataset.click(function() {
         if (confirm("Are you sure you want to delete all records for this dataset?")) {
             $.ajax({
-                url: Routing.generate("pelagos_api_datasets_delete", { id: deleteDataset.attr("datasetId") }),
+                url: Routing.generate("pelagos_api_datasets_get",{ "id" : deleteDataset.attr("datasetId") }),
```

Comment:
? this is weird

Reply:
tldr; and this is why we always use routes, and no the absolute address.